### DISCUSSION TITLE
RFC: context-aware version dropdown for multi-instance setups

### DISCUSSION THREAD

**Discussion by ZenaMel** [facebook/docusaurus#8528] [Ideas]
************************************* DRAFT *************************************

# Motivation

This RFC is a natural evolution from discussions on Discord with me, @slorber and @homotechsual.
See: https://discordapp.com/channels/398180168688074762/867060369087922187/1060875802801098793 to follow the prior discussion.

# Use-case scenario

- You have several products to document, each with their own release cycle.
- You'd like to synchronize the docs to each product cycle.

# Current solution and problems

To illustrate the problem with just using multi-instance versioning out-of-the-box:

... (to be continued.)

**Comment by ZenaMel** [comment]
I'll keep working on this in the following weeks/months. I hope it's fine for me to use this as a draft space and jot down my thoughts until I've formulated a complete, potential design solution.

**Reply by slorber** [reply]
ok for me

seems related to https://github.com/facebook/docusaurus/issues/4389
